Dockerfile

```diff
@@ -1,5 +1,15 @@
-# govukpay/nodejs:alpine-3.8.1
-FROM govukpay/nodejs@sha256:a3ae1b0b3ab5e7cdefe0787b97b1047cdc5e0c5166fb677e3cefac538bc99f86
+# alpine:3.8
```

Comment:
The current base image pull from alpine 3.9 -- just making sure this regression is intentional

Reply:
The current code on `pay-infra` `master` does, but that's not the code which generated `govukpay/nodejs:alpine-3.8.1`

Alpine 3.9 includes node 10 and appmetrics fails to `npm install` on that for some reason I haven't got to the bottom of yet